Source/UI/Windows/SpeakerSetup/SpeakerGroupSettingsWindow.cpp

```diff
@@ -39,7 +39,9 @@ SpeakerGroupSettingsComponent::SpeakerGroupSettingsComponent(SpeakerGroupCompone
     };
     auto initSlider = [this](juce::Slider & slider, float value) {
         slider.setRange (0, 360.0, 0.1);
+#if USE_JUCE_8
```

Comment:
what about

#if JUCE_VERSION >=  0x80000

so that it's not confusing in five years when it's JUCE 9 time ?